app/main.py

```diff
@@ -10,25 +12,47 @@
 from app.api.api_v1 import api_router
 from app.core.config import settings, DevPhase
 from app.core.middleware import AuthenticationMiddleware, AuthBackend, LogMiddleware
-from eduhelx_utils.custom_logger import CustomizeLogger
 from app.core.exceptions import CustomException
 
 import logging
 from pathlib import Path
 
+
+formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
+
+def init_logging():
+    logging.getLogger("uvicorn.access").disabled = True
+    root_logger = logging.getLogger()
+    root_logger.disabled = True
+    for handler in root_logger.handlers[:]:
+        root_logger.removeHandler(handler)
+
+def init_uvicorn_logger(formatter) -> logging.Logger:
```

Comment:
We weren't setting the formatter in the logging_config.json, we were only setting the log level and the log retention. Now we're just using a rotating file handler so it once a log file gets to be large enough (~1MB) it will save as an archive backup and start logging to a new one, for up to 4 backups. And the log level could probably be changed based on `DEV_PHASE`, so I could do that if you want, but I don't think we need a logging_config.json anymore personally.